src/sentry/static/sentry/app/utils/statics-setup.tsx

```diff
@@ -4,11 +4,13 @@
 declare var __webpack_public_path__: string;
 
 /**
- * Set the webpack public path at runtime. The __sentryGlobalStaticPrefix will
- * be declared in layout.html.
+ * Set the webpack public path at runtime. This is necessary so that imports can be resolved properly
  *
  * NOTE: This MUST be loaded before any other app modules in the entrypoint.
+ *
+ * XXX(epurkhiser): Currently we only boot with hydration in experimental SPA
+ * mode, where assets are *currently not versioned*. We hardcode `/_assets/` here
+ * for now as a quick workaround for the index.html being aware of versioned
+ * asset paths.
  */
-if (window.__sentryGlobalStaticPrefix) {
-  __webpack_public_path__ = window.__sentryGlobalStaticPrefix;
-}
+__webpack_public_path__ = window.__initialData?.distPrefix || '/_assets/';
```

Comment:
We will be able to simplify this even further after the django manifest work is done, as we can then remove versioning from URL path